Using fqdn to create the CNAME file doesn't work

I have this configuration:
```yaml
jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v1
      - name: Deploy GitHub Pages
        uses: crazy-max/ghaction-github-pages@v1
        with:
          build_dir: public
          fqdn: example.com
          target_branch: gh-pages
        env:
          # GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          # https://github.community/t5/GitHub-Actions/Github-action-not-triggering-gh-pages-upon-push/td-p/26869/
          GITHUB_PAT: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
and I'm getting the following error:
```bash
✅ Use GITHUB_PAT
🏃 Initializing local git repo
git init .
Initialized empty Git repository in /tmp/github-pages-p0j0xG/.git/
git checkout --orphan gh-pages
Switched to a new branch 'gh-pages'
🏃 Copying /home/runner/work/repo/repo/public contents to /tmp/github-pages-p0j0xG
✍️ Writing example.com domain name to public/CNAME
##[error]ENOENT: no such file or directory, open 'public/CNAME'
##[error]Node run failed with exit code 1
```

I'm pretty sure that `path.join(build_dir, 'CNAME')` (https://github.com/crazy-max/ghaction-github-pages/blob/master/src/main.ts#L59) needs to be `path.join(tmpdir, 'CNAME')`

--- a/lib/main.js
+++ b/lib/main.js
@@ -69,8 +69,8 @@ function run() {
             core.info(`🏃 Copying ${path.join(currentdir, build_dir)} contents to ${tmpdir}`);
             fs_extra_1.copySync(path.join(currentdir, build_dir), tmpdir);
             if (fqdn) {
-                core.info(`✍️ Writing ${fqdn} domain name to ${path.join(build_dir, 'CNAME')}`);
-                fs.writeFileSync(path.join(build_dir, 'CNAME'), fqdn.trim());
+                core.info(`✍️ Writing ${fqdn} domain name to ${path.join(tmpdir, build_dir, 'CNAME')}`);
+                fs.writeFileSync(path.join(tmpdir, build_dir, 'CNAME'), fqdn.trim());
             }
             core.info(`🔨 Configuring git committer to be ${committer_name} <${committer_email}>`);
             yield exec.exec('git', ['config', 'user.name', committer_name]);

--- a/lib/main.js
+++ b/lib/main.js
@@ -69,8 +69,8 @@ function run() {
             core.info(`🏃 Copying ${path.join(currentdir, build_dir)} contents to ${tmpdir}`);
             fs_extra_1.copySync(path.join(currentdir, build_dir), tmpdir);
             if (fqdn) {
-                core.info(`✍️ Writing ${fqdn} domain name to ${path.join(tmpdir, build_dir, 'CNAME')}`);
-                fs.writeFileSync(path.join(tmpdir, build_dir, 'CNAME'), fqdn.trim());
+                core.info(`✍️ Writing ${fqdn} domain name to ${path.join(tmpdir, 'CNAME')}`);
+                fs.writeFileSync(path.join(tmpdir, 'CNAME'), fqdn.trim());
             }
             core.info(`🔨 Configuring git committer to be ${committer_name} <${committer_email}>`);
             yield exec.exec('git', ['config', 'user.name', committer_name]);

--- a/src/main.ts
+++ b/src/main.ts
@@ -55,8 +55,8 @@ async function run() {
     copySync(path.join(currentdir, build_dir), tmpdir);
 
     if (fqdn) {
-      core.info(`✍️ Writing ${fqdn} domain name to ${path.join(build_dir, 'CNAME')}`);
-      fs.writeFileSync(path.join(build_dir, 'CNAME'), fqdn.trim());
+      core.info(`✍️ Writing ${fqdn} domain name to ${path.join(tmpdir, build_dir, 'CNAME')}`);
+      fs.writeFileSync(path.join(tmpdir, build_dir, 'CNAME'), fqdn.trim());
     }
 
     core.info(`🔨 Configuring git committer to be ${committer_name} <${committer_email}>`);

--- a/src/main.ts
+++ b/src/main.ts
@@ -55,8 +55,8 @@ async function run() {
     copySync(path.join(currentdir, build_dir), tmpdir);
 
     if (fqdn) {
-      core.info(`✍️ Writing ${fqdn} domain name to ${path.join(tmpdir, build_dir, 'CNAME')}`);
-      fs.writeFileSync(path.join(tmpdir, build_dir, 'CNAME'), fqdn.trim());
+      core.info(`✍️ Writing ${fqdn} domain name to ${path.join(tmpdir, 'CNAME')}`);
+      fs.writeFileSync(path.join(tmpdir, 'CNAME'), fqdn.trim());
     }
 
     core.info(`🔨 Configuring git committer to be ${committer_name} <${committer_email}>`);

